apps/reviews/templates/reviews/review.html

```diff
@@ -85,13 +85,13 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
     {% if review.user_id == request.user.id %}
       <li>
         <a class="review-edit" href="#">
-          {{ _('Edit review') }}</a>
+          {{ _('Edit reply') }}</a>
```

Comment:
You need to make sure that it only says "reply" if it's a reply, not in all the cases. If you check a few lines above (on line 79), there's an example use of `is_reply`.

Using this, you should be able to display "reply" or "review" when it's relevant.

Let me know if you need more help!